completions/bash/podman

```diff
@@ -1524,6 +1524,56 @@ _podman_info() {
     esac
 }
 
+_podman_image_umount() {
+     _podman_image_unmount
+}
+
+_podman_image_unmount() {
+    local boolean_options="
+	--all
+	-a
+	--help
+	-h
+	--force
+	-f
+    "
+    local options_with_args="
+    "
+
+    local all_options="$options_with_args $boolean_options"
+    case "$cur" in
+	-*)
+	    COMPREPLY=($(compgen -W "$boolean_options $options_with_args" -- "$cur"))
+	    ;;
+	*)
+	    __podman_complete_images --force-tag --id
```

Comment:
As an enhancement, would be great to only complete those images that are mounted, not all of them. Can be addressed as a followup.

Reply:
Agree.